hcloud/schema.go

```diff
@@ -124,7 +124,7 @@ func DatacenterFromSchema(s schema.Datacenter) *Datacenter {
 }
 
 // ServerFromSchema converts a schema.Server to a Server.
-func ServerFromSchema(s schema.Server) *Server {
```

Comment:
There is no need to add the `ServersClient` here.

Reply:
I added this for use 2 API key or more 
I need this but if you have a better solution say

Reply:
A better solution would be to just use two client instances.

Reply:
`var servers = make(map[string]*hcloud.Server)`
I have this but I can't reboot or sth else with this
I added `ServerClient` to `Server` for this reason
but ok I delete that for merge console request
if you have any idea for use 2 or more API key plz say

Reply:
You could just use two Client instances. In our mind, it does not make sense to add a client here:
```go
client1 := hcloud.NewClient(hcloud.WithToken("token1"))
client2 := hcloud.NewClient(hcloud.WithToken("token2"))
```